lib/Doctrine/ruleset.xml

```diff
@@ -202,6 +202,14 @@
     <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
     <!-- Forbid assignments in conditions -->
     <rule ref="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
+    <!-- Require consistent spacing for control structures -->
+    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
```

Comment:
Are the @doctrine/coding-standard-approvers team okay of approving one by one of this Sniff in particular? 'cause I'd like to propose some new lines after some tokens in specific, as well :)

Reply:
You can send a cumulative patch, it just lowers the chance of merge due to more possible discussions

Reply:
This. Originally I wanted to introduce this with default settings, but it requires spaces around yields, which night be controversial and I don't like it personally either